### PR TITLE
extended the SDK not found exception

### PR DESCRIPTION
**Pull request by demanbart**
Based upon issue 108 I've added a small alteration to the setup.py scripts which can help people using pybluez combined with windows10

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -43,12 +51,16 @@ def find_MS_SDK():
             candidate_sdk = os.path.join(candidate_root, candidate_path)
             if os.path.exists(candidate_sdk):
                 return candidate_sdk
-
+    raise SDKException("Could not find the Windows Platform SDK, for python {0}.{1} consider installing Visual Studio {2}".format(sys.version_info.major,
```
Comment (karulis) [inline setup.py]: Line too long

Comment (demanbart) [reply setup.py]: Noted and will apply changes

**Comment by rgov** [comment]
Since this is a major source of complaints, and @karulis's review was rather short, I'll merge this, but I do not have a way of testing it myself. Thanks for the contribution.

File: setup.py [hunk 2]
```diff
@@ -31,13 +50,10 @@ def find_MS_SDK():
...
     if sys.version < '3.3':
         candidate_paths.append(r'Microsoft SDKs\Windows\v6.0A')  # Visual Studio 9
-        vs_version = 9
     elif '3.3' <= sys.version < '3.5':
         candidate_paths.append(r'Microsoft SDKs\Windows\v7.0A')  # Visual Studio 10
-        vs_version = 10
     elif sys.version >= '3.5':
         candidate_paths.append(r'Microsoft SDKs\Windows\v10.0A')  # Visual Studio 14
-        vs_version = 14
     else:
```
Comment (karulis) [inline setup.py]: This will never happen

Comment (karulis) [inline setup.py]: This else will never happen, can be removed

**Pull request merged**